lib/gtfs/components/gtfs-search.js

```diff
@@ -85,8 +85,10 @@ const _sortOptions = (a: GtfsOption, b: GtfsOption, input: string) => {
  * GTFS stops and routes using the user-inputted text.
  */
 class GtfsSearch extends Component<Props, State> {
-  state = {
-    value: this.props.value
+  componentWillMount () {
```

Comment:
These thingss are in here because I needed mastarm 4.4.1 for some reason.  There were some babel changes in mastarm that made the initialization of state from props impossible, so I refactored that.  Perhaps they should be refactored to be part of a constructor since componentWillMount will be removed in React 17.

Reply:
Want me to break out the update to mastarm 4.4.1 in another PR?

Reply:
No, that's ok. I was just struggling to understand the reason for all of these changes. Could you create an issue documenting some of the changes that will need to happen in preparation for a future React update so we can track this?

Reply:
Created #253